### ISSUE TITLE
Create dataset must return a train, val and test set

### ISSUE THREAD

**Issue by Gavriel94** [Gavriel94/ecoacoustics-pix2pix#65]
To reconstruct a spectrogram into audio you need details such as magnitude, phase, number of fast Fourier transforms and more. Magnitude and phase are each a large number of complex values which require lots of disk space. Each JSON file is between 168-174MB. The test set will be used to evaluate the model so a params dict will only be generated for a test set member. This means the data has to be organised into train/val/test before the training script runs.
